File: DSA/Pragya_Agarwal/Day24/Day24_Question1.py
class Solution:
    def sortedSquares(self, nums: List[int]) -> List[int]:
       N = len(nums)                
       result = [0] * N                
       for i in range(N):             
        result[i] = nums[i] * nums[i]  
            
       result.sort()                   
       return result

File: DSA/Arshdeep-Kaur/Day 24/Day24_Question1.cpp
class Solution {
public:
    vector<int> sortedSquares(vector<int>& nums) {
        vector<int> ans;

        for (int i:nums) {
            int temp = i*i;
            ans.push_back(temp);
        }
        sort(ans.begin(), ans.end());

        return ans;
    }
};
